x-pack/legacy/plugins/canvas/public/components/text_style_picker/text_style_picker.js

```diff
@@ -9,8 +9,11 @@ import PropTypes from 'prop-types';
 import { EuiFlexGroup, EuiFlexItem, EuiSelect, EuiSpacer, EuiButtonGroup } from '@elastic/eui';
 import { FontPicker } from '../font_picker';
 import { ColorPickerPopover } from '../color_picker_popover';
+import { ComponentStrings } from '../../../i18n';
 import { fontSizes } from './font_sizes';
 
+const { TextStylePicker: strings } = ComponentStrings;
```

Comment:
I have the translation for this component here: https://github.com/elastic/kibana/pull/46981